testsuite/integration-arquillian/tests/other/console/src/main/resources/themes/qe/login/messages/messages_en.properties

```diff
@@ -0,0 +1,222 @@
+doLogIn=Log in
```

Comment:
Only include the message you are overwriting, the other messages are inherited from the parent theme.